Support Gatsby v3

Gatsby v3 is officially out - https://github.com/gatsbyjs/gatsby/issues/29932. While migrating to v3 I'm getting error while running `npm install`:

> npm ERR! peer dep missing: gatsby@^2.20.0, required by gatsby-remark-embedder@4.1.0

Is there a way to test this plugin against gatsby v3?

I will be more than happy to submit a PR 🚀

--- a/.github/workflows/validate.yml
+++ b/.github/workflows/validate.yml
@@ -16,7 +16,7 @@ jobs:
     if: ${{ !contains(github.head_ref, 'all-contributors') }}
     strategy:
       matrix:
-        node: [10.13, 10, 12, 14]
+        node: [12.13, 12, 14, 15]
     runs-on: ubuntu-latest
     steps:
       - name: 🛑 Cancel Previous Runs

--- a/package.json
+++ b/package.json
@@ -62,10 +62,10 @@
     "remark-html": "^13.0.1"
   },
   "peerDependencies": {
-    "gatsby": "^2.20.0"
+    "gatsby": "^3.0.0"
   },
   "engines": {
-    "node": ">=10.13.0",
+    "node": ">=12.13.0",
     "npm": ">=6",
     "yarn": ">=1"
   }

